pkg/operator/ceph/file/mds/livenessprobe.sh

```diff
@@ -7,7 +7,8 @@ FILESYSTEM_NAME="{{ .FilesystemName }}"
 KEYRING="{{ .Keyring }}"
 CMD_TIMEOUT="{{ .CmdTimeout }}"
 
-outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --keyring "$KEYRING" --connect-timeout="$CMD_TIMEOUT" --format json)"
+export CEPH_ARGS="--keyring $KEYRING" # workaround till ceph does not fix the issue with keyring
```

Comment:
Let's discuss this. It's clearly a bug that the `ceph` CLI command isn't accepting the `--keyring` arg. Let's try to figure out why and if that can be fixed.